xcodeproj/internal/xcodeproj_rule.bzl

```diff
@@ -448,6 +448,12 @@ actual targets: {}
         dest_label = bazel_labels.normalize_label(dest_target.label)
         if src_label in unfocused_labels or dest_label in unfocused_labels:
             continue
+
+        # Exclude targets not in either focused or unfocused targets from
+        # potential merges since they're excluded from Xcode.
+        merge_src_is_xcode_target = merge.src.id in focused_targets or merge.src.id in unfocused_targets
```

Comment:
Do you dislike that the comment isn't necessarily true to the goal or the check itself? Like would this help?
```suggestion
        # Exclude targets not in either focused or unfocused targets from
        # potential merges since they're not possible Xcode targets.
```

Reply:
Ohh, the name is off I think? (I wrote this stuff and it's do confusing me).

Reply:
I'm mainly concerned with us saying "unfocused targets make Xcode targets" which is what is implied by the check itself. I'm not at my computer today, but what is the difference between unfocused_labels and unfocused_targets? Maybe that's my confusion (and something needs renaming).